.github/workflows/integration.yaml

```diff
@@ -55,13 +55,40 @@ jobs:
         run: |
           docker load --input /tmp/ethrex_image.tar
 
-      - name: Setup kurtosis testnet and run assertoor tests
+      - name: Run assertoor
         uses: ethpandaops/kurtosis-assertoor-github-action@v1
         with:
+          enclave_name: "ethrex-assertoor-tx"
           kurtosis_version: "1.4.2"
           ethereum_package_url: "github.com/lambdaclass/ethereum-package"
           ethereum_package_branch: "ethrex-integration"
-          ethereum_package_args: "./test_data/network_params.yaml"
+          ethereum_package_args: "./test_data/assertoor/network_params_tx.yaml"
+
+  run-assertoor-blob-check:
+    name: Assertoor - Blobs & Stability Check
+    runs-on: ubuntu-latest
+    needs: [docker_build]
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Download artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: ethrex_image
+          path: /tmp
+
+      - name: Load image
+        run: |
+          docker load --input /tmp/ethrex_image.tar
+
+      - name: Run assertoor
+        uses: ethpandaops/kurtosis-assertoor-github-action@v1
+        with:
+          enclave_name: "ethrex-assertoor-blob"
+          kurtosis_version: "1.4.2"
+          ethereum_package_url: "github.com/lambdaclass/ethereum-package"
+          ethereum_package_branch: "ethrex-integration"
+          ethereum_package_args: "./test_data/assertoor/network_params_blob.yaml"
```

Comment:
can we move the ci specific files to be under `.github/something`? `.github/config` maybe? We already have `.github/scripts`

Reply:
Added them inside `.github/config/assertoor`